pkgs/development/python-modules/python-nest/default.nix

```
@@ -0,0 +1,28 @@
+{ buildPythonPackage, fetchFromGitHub, lib, python, python-dateutil, requests
+, six, sseclient-py }:
+
+buildPythonPackage rec {
+  pname = "python-nest";
+  version = "4.1.0";
+
+  # pypi does not contain tests, using github sources instead
+  src = fetchFromGitHub {
+    owner = "jkoelker";
+    repo = "python-nest";
+    rev = "b45100440577ca35061e08dda48fddcc49f214eb";
```

Let's use the tag here.

```suggestion
    rev = "v${version}";
```